### PR TITLE
Add symbols for thorn, eth, …

### PR DESCRIPTION
**Pull request by czeckd**
Add symbols for thorn, eth, minus, 3/4, dagger, heart, diamond, club, and spade

### CODE REVIEW

File: symbols.js [hunk 1]
```diff
@@ -54,6 +54,11 @@ const symbols = [
         name: "Quarter",
         searchTerms: ["quarter", "1/4"]
     },
+    {
+        glyph: "¾",
+        name: "Three-Quarter",
+        searchTerms: ["three", "quarter", "3/4"]
+    },
```
Comment (qwell) [inline symbols.js:61]: Added in #43.

File: symbols.js [hunk 2]
```diff
@@ -64,6 +69,11 @@ const symbols = [
         name: "Plus-minus",
         searchTerms: ["plus", "minus"],
     },
+    {
+        glyph: "−",
+        name: "minus",
+        searchTerms: ["minus"],
+    },
```
Comment (qwell) [inline symbols.js:76]: Are there seriously keyboards without a minus symbol?  What about plus?

File: symbols.js [hunk 3]
```diff
@@ -287,6 +297,51 @@ const symbols = [
...
+    {
+        glyph: "þ",
+        name: "thorn",
+        searchTerms: ["thorn"],
+    },
+    {
+        glyph: "Þ",
+        name: "thorn",
+        searchTerms: ["thorn"],
+    },
```
Comment (qwell) [inline symbols.js:309]: These should specify uppercase and lowercase.

File: symbols.js [hunk 4]
```diff
@@ -287,6 +297,51 @@ const symbols = [
...
+    {
+        glyph: "ð",
+        name: "eth",
+        searchTerms: ["eth"],
+    },
+    {
+        glyph: "Ð",
+        name: "eth",
+        searchTerms: ["eth"],
+    },
```
Comment (qwell) [inline symbols.js:319]: Also specify uppercase and lowercase.

**Review by qwell — CHANGES_REQUESTED** [review]
Note too that searchTerms will soon include the glyph and the name, so they shouldn't be there in most cases.

File: symbols.js (hunk 2 above)
Comment (czeckd) [reply symbols.js:76]: The keyboard - (dash or hyphen) is different typologically from the − minus.

File: symbols.js (hunk 1 above)
Comment (czeckd) [reply symbols.js:61]: Great! I will remove from mine.

**Comment by czeckd** [comment]
> Note too that searchTerms will soon include the glyph and the name, so they shouldn't be there in most cases.

Should I then have empty searchTerms then if they are duplicates?

**Comment by qwell** [comment]
You can just remove searchTerms entirely unless there are terms that wouldn't be in the name field.  All of these, except for minus (which could have searchTerms: "-" for example) could just remove it.

**Comment by czeckd** [comment]
Okay. Hopefully, this is correct now.

**Review by qwell — APPROVED** [review]

**Comment by samwho** [comment]
Thanks @czeckd for your contribution! 🙏

**Pull request merged**